utils/db_process.py

```diff
@@ -0,0 +1,93 @@
+import mysql.connector
+from mysql.connector import pooling
+import os
+from dotenv import load_dotenv
+from typing import Union, Tuple, Dict, List
+from typing import Final
+import datetime
+load_dotenv()
+
+pool_config = {
+    "pool_name": "pool",
+    "pool_size": 5,
+    "host": os.getenv("DB_HOST"),
+    "user": os.getenv("DB_USER"),
+    "password": os.getenv("DB_PASSWORD"),
+    "database": os.getenv("DB_DATABASE"),
+}
+pool_config: Final
```

Comment:
'Final' name should be initialized with a value. [Ref](https://peps.python.org/pep-0591/#syntax)